spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractServiceRegistrationBean.java

```diff
@@ -131,6 +150,134 @@ public final U setDecorators(
         return self();
     }
 
+    /**
+     * Returns example requests of {@link #getService()}.
+     */
+    @NotNull
+    public Collection<V> getExampleRequests() {
+        return exampleRequests;
+    }
+
+    /**
+     * Sets example requests for {@link #getService()}.
+     */
+    public U setExampleRequests(@NotNull Collection<V> exampleRequests) {
+        this.exampleRequests = exampleRequests;
+        return self();
+    }
+
+    /**
+     * Sets example requests for {@link #getService()}.
+     */
+    public U setExampleRequests(@NotNull Iterable<V> exampleRequests) {
+        return setExampleRequests(ImmutableList.copyOf(exampleRequests));
+    }
+
+    /**
+     * Sets example requests for {@link #getService()}.
+     */
+    public U setExampleRequests(@NotNull V... exampleRequests) {
+        return setExampleRequests(ImmutableList.copyOf(exampleRequests));
+    }
+
+    /**
+     * Adds example requests for {@link #getService()}.
+     */
+    public U addExampleRequests(@NotNull Collection<V> exampleRequests) {
+        this.exampleRequests = ImmutableList.<V>builder().addAll(this.exampleRequests)
+                                                         .addAll(exampleRequests)
+                                                         .build();
+        return self();
+    }
+
+    /**
+     * Adds example requests for {@link #getService()}.
+     */
+    public U addExampleRequests(@NotNull Iterable<V> exampleRequests) {
```

Comment:
- `? extends V`
- I think we need to be consistent about delegation, like always delegating from `Collection` to `Iterable`:
  - `addExampleRequests(Collection)` delegates to `addExampleRequests(Iterable)`
  - `setExampleRequests(Collection)` delegates to `setExampleRequests(Iterable)`
  - (Same for example headers)